src/Compilers/Core/Desktop/CommandLine/ErrorLogger.Issue.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License; Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal partial class ErrorLogger
+    {
+        /// <summary>
+        /// Represents an issue to be logged into the error log.
+        /// This could be corresponding to a <see cref="Diagnostic"/> or a <see cref="DiagnosticInfo"/> reported by the <see cref="CommonCompiler"/>.
```

Comment:
Nope, they are different, see http://source.roslyn.io/Microsoft.CodeAnalysis/R/f445789e05e0e7b5.html

Although, we can probably just invoke this method: http://source.roslyn.io/#Microsoft.CodeAnalysis/Diagnostic/Diagnostic.cs,bf215ec6ef08f6fc,references and convert DiagnosticInfo to a Diagnostic with NoLocation. I'll make that change to avoid 2 methods for this.